front/src/containers/SearchPage/SearchPage.tsx

```diff
@@ -732,13 +732,11 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
     const searchQueryString = new URLSearchParams(
       this.props.history.location.search
     );
-    //console.log("SITE PARAMS", this.props.site);
     const siteViewUrl = searchQueryString.getAll('sv').toString() || 'default';
-    // This assumes that the site provider is not the passing a url into the page
+    // This assumes that the site provider is not passing a url into the page
     // view fragment portion of the query otherwise we would need to call the
     //  page view query without passing the url into it to retrieve the default url
-    // @ts-ignored
-    const defaultPageView = this.props.site.pageView.url;
+    const defaultPageView = this.props.site.pageView!.url;
```

Comment:
@rodctech hey, just a note about the ! here.  The server is trying to say the pageView might not be there which is true if there are no pages (or if there's no default).  I get that we don't need to deal with this in an elegant way since *most* of the time there will be a page - however - if you get a typescript error like this one it's a good indication to stop and think about why and how we can make life better for the next guy to come though.  For example, what if we used ?. instead of !. here and then put an if (!defaultPageView) { console.error("Hey, you might need to add a page view or something") } or even `throw 'need to make a page'`.  This reminder makes it easier for new contributors to run the site and it makes it easier for us when setting up a new system because we won't have to remember/debug why the site just goes white when we try to click a study.